README.md

```diff
@@ -5,7 +5,7 @@ An official channel to discuss the development of this source port can be found
 
 # Features
 - Hardware accelerated GPU rendering
-- Native 960x544 resolution
+- Native 1920x1080 resolution (with [Sharpscale](https://forum.devchroma.nl/index.php/topic,112.0.html))
```

Comment:
Revert this line in the readme since misleading (native Vita resolution is still 960x544). If anything add a new line specifying high res is supported for PSTV. Also, if you want to link SharpScale, use a direct download link or either a git repo link (assuming you have a public repo for it with source and everything).